src/electionguard_cli/e2e_steps/input_retrieval_step.py

```diff
@@ -15,10 +15,6 @@
 class InputRetrievalStep(E2eStepBase):
     """Responsible for retrieving and displaying user provided inputs."""
 
-    def get_ballots(self) -> List[PlaintextBallot]:
```

Comment:
The issue with these is all the same. They are not using `self` or using anything internal to the class. With the way python is structured everything is public, it doesn't make sense for this embedding and it makes complicated name structures. Almost all linters bark at this.